client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -245,37 +251,10 @@ class VerificationPortalFormClass extends Component {
 						<Field name="verifiedData.services.verified" component={Toggle} />
 					</div>
 				</div>
-
 				<div className={classes.formSection}>
 					<Heading text="HOURS" size="medium" />
 					<Spacer height="50px" />
-					<label>Sunday Hours</label>
-					<Field name="hours[0].open" component={Time} />
-					<Field name="hours[0].close" component={Time} />
-
-					<label>Monday Hours</label>
-					<Field name="hours[1].open" component={Time} />
-					<Field name="hours[1].close" component={Time} />
-
-					<label>Tuesday Hours</label>
-					<Field name="hours[2].open" component={Time} />
-					<Field name="hours[2].close" component={Time} />
-
-					<label>Wednesday Hours</label>
-					<Field name="hours[3].open" component={Time} />
-					<Field name="hours[3].close" component={Time} />
-
-					<label>Thursday Hours</label>
-					<Field name="hours[4].open" component={Time} />
-					<Field name="hours[4].close" component={Time} />
-
-					<label>Friday Hours</label>
-					<Field name="hours[5].open" component={Time} />
-					<Field name="hours[5].close" component={Time} />
-
-					<label>Saturday Hours</label>
-					<Field name="hours[6].open" component={Time} />
-					<Field name="hours[6].close" component={Time} />
+					<Day hours={hours} />
```

Comment:
@bhavyaab What do you think of naming this component `Days` since it displays all 7 days?

Reply:
What about BusinessHours?